paclair/docker/docker_image.py

```diff
@@ -42,6 +42,15 @@ def token(self):
         """
         return self.registry.get_token(self)
 
+    @property
+    def token_type(self):
```

Comment:
Maybe instead of adding a new property token_type, you should change token to a property called authorization which should call registry.get_authorization()

Reply:
done.